app/src/main/res/layout/fragment_chat_rooms.xml

```diff
@@ -8,6 +8,7 @@
 
     <androidx.recyclerview.widget.RecyclerView
         android:id="@+id/recycler_view"
+        android:layout_below="@+id/connection_status_text"
```

Comment:
Should be `text_connection_status` instead.

Reply:
`id/connection_status_text` was defined earlier should i rename it.

Reply:
Oh, I'm sorry! If you can, please change it then!
Thanks @shubhsherl 👍